source/braille.py

```diff
@@ -1211,6 +1213,20 @@ def _set_windowEndPos(self, endPos):
 		if region.focusToHardLeft:
 			# Only scroll to the start of this region.
 			restrictPos = endPos - regionPos - 1
+		elif config.conf["braille"]["focusContextPresentation"]==2:

```

Comment:
While NVDA certainly isn't too strict about this, I think having constants for this option would improve readability a lot.